src/KronPhotometry.cc

```diff
@@ -265,7 +257,8 @@ PTR(KronAperture) KronAperture::determineRadius(
         //
         // Build an elliptical Footprint of the proper size
         //
-        afw::detection::Footprint foot(afw::geom::ellipses::Ellipse(axes, center));
+        afw::detection::Footprint foot(afw::geom::SpanSet::fromShape(
+            afw::geom::ellipses::Ellipse(axes, center)));
```

Comment:
A note here, we have discussed this a bit in person, but a note here for posterity. The API for footprint was chosen to only take a shared pointer to a SpanSet such that 1. Code duplication was minimized (maintaining two similar parallel apis) 2. To encourage developers to used a SpanSet instead of a footprint whenever possible. These two things were weighted against verbosity, and ultimately we decided against verbosity of code